src/test/java/com/google/cloud/spanner/pgadapter/ITJdbcDescribeStatementTest.java

```diff
@@ -210,9 +214,29 @@ public void testParameterMetaData() throws SQLException {
     }
   }
 
+  @Test
+  public void testManualParameters() throws SQLException {
+    LOGGER.info("Running test");
```

Comment:
Are these lines required?

Reply:
No, I was doing some testing against a locally built version and needed some more logging. I'll remove this before merging.